dartagnan/src/main/java/com/dat3m/dartagnan/program/event/lang/linux/LKMMLoad.java

```diff
@@ -8,8 +8,13 @@
 
 public class LKMMLoad extends Load {
```

Comment:
This is because LKMMLoad is unforunately not compiled when verifying under LKMM, so it has to be a core event.
If I add compilation to a normal Load, we lose the custom printing as `READ_ONCE` after processing.
If you are willing to lose the custom printing, I can make it a non-core event. If you want to keep the custom printing, you will need to wait for a follow-up PR that fixes this :).

Reply:
More precisely, it will be a `LoadBase`, it will always get compiled to a core-level `Load` and if compiled to Linux, we add custom printing.
Keep in mind that `SingleAccessMemoryEvent` and `LoadBase` are just classes to share implementation, so it would also be feasible (and semantically identical) to have `LKMMLoad` inherit from neither of those.